website/source/docs/platform/k8s/run.html.md

```diff
@@ -43,59 +46,122 @@ the [recommended security features](/docs/internals/security.html). Currently,
 some of these features are not supported in the Helm chart and require additional
 manual configuration.
 
-## How-To
+### Prerequisites
+
+The Consul Helm chart works with Helm 2 and Helm 3. If using Helm 2, you will
+need to install Tiller by following the
+[Helm 2 Installation Guide](https://v2.helm.sh/docs/using_helm/#quickstart-guide).
 
 ### Installing Consul
 
-To install Consul, clone the consul-helm repository, checkout the latest release, and install
-Consul. You can run `helm install` with the `--dry-run` flag to see the
-resources it would configure. In a production environment, you should always
-use the `--dry-run` flag prior to making any changes to the Consul cluster
-via Helm.
+Determine the latest version of the Consul Helm chart
+by visiting [https://github.com/hashicorp/consul-helm/releases](https://github.com/hashicorp/consul-helm/releases).
 
-```sh
-# Clone the chart repo
-$ git clone https://github.com/hashicorp/consul-helm.git
-$ cd consul-helm
+Clone the chart at that version, for example if the latest version is
+`v0.8.1` you would run:
 
-# Checkout a tagged version
-$ git checkout v0.1.0
+```bash
```

Comment:
I moved these instructions over from the helm.html page. I think they're a bit cleaner